_posts/2023-04-04-specify-http-digest-algorithms.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Configure the digest authentication mechanism to accept SHA-256 and SHA-512-256 algorithms'
```

Comment:
Maybe could reword to "Configuring the DIGEST mechanism to make use of additional algorithms"

Reply:
fixed. Thanks!